src/microbe_stage/species_component.h

```diff
@@ -35,6 +35,8 @@ class SpeciesComponent : public Leviathan::Component {
     std::string epithet;
     double aggression;
     double fear;
+	int32_t population;
+	int32_t generation;
```

Comment:
There's also tabs here. You should run the formatting script `RunCodeFormatting.rb` (make sure to have clang 6.0 in PATH) to fix all of the tab issues at once.